Makefile

```diff
@@ -1,6 +1,12 @@
 PHP_CS_FIXER_FUTURE_MODE=1
 PHPSTAN=./phpstan.phar
 PHP-CS-FIXER=./php-cs-fixer-v2.phar
+PHPUNIT=vendor/bin/phpunit
+
+DOCKER_RUN=docker run -t --rm -v "$$PWD":/opt/infection -w /opt/infection
+DOCKER_RUN_70=flock devTools/*php70*.json $(DOCKER_RUN) infection_php70
```

Comment:
could you please explain why we need `flock` here?

Reply:
`make -j` will start `docker run` for all targets at once, it will be like if you start `phpunit` multiple times at once. I believe we can't run tests in parallel now because of some issues, which may not be worth fixing. Hence we have to restrict Make from running PHP-version specific targets, which use the same VM, in parallel.